src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Lottery {
+
+    private static ColorSupplier colorSupplier = new ColorSupplier();
+
+    public static Ball getRandomBall() {
```

Comment:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#dont-use-static-methods-in-your-solution